style.css

```diff
@@ -50,3 +50,85 @@ form {
 .form-h2 {
   width: 260px;
 }
+
+#list-allbooks {
+  display: none;
+  margin-top: 5rem;
+}
+
+#contact-box {
+  display: none;
+}
+
+/* Header styling */
+
+nav {
+  font-family: sans-serif;
+  width: 90%;
+  margin: 1rem auto;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  border: solid 3px black;
+  padding: 1rem;
+}
```

Comment:
- [ ] Kindly adhere to the project requirement because as I can see your `header` navigation have too much space around it, which makes your design `inconsistent` and looks unprofessional because the *navigation-menu* is also not aligned properly with your awesomeBook logo. Please have a look at the below-attached image and table.

### Design Comparison
<table>
<tr>
<th>Given Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![image](https://github.com/blessing-michael/books-claases/assets/66476329/5fa6479a-dd7f-403e-829d-3b262d85fdcf)

</td>
<td>

![Screenshot from 2023-08-12 21-20-05](https://github.com/blessing-michael/books-claases/assets/66476329/35e8de7d-98e7-48f0-872f-88a9dd15f9c0)

</td>
</tr>
</table>

Reply:
Fixed